pxr/base/tf/mallocTag.cpp

```diff
@@ -190,8 +190,8 @@ Tf_MallocTagStringMatchTable::SetMatchList(const std::string& matchList)
 {
     _matchStrings.clear();
     std::vector<std::string> items = TfStringTokenize(matchList, ",\t\n");
-    TF_FOR_ALL(i, items) {
-        _matchStrings.push_back(_MatchString(TfStringTrim(*i, " ")));
+    for(std::string i: items) {
```

Comment:
could be a reference here, `std::string&`. could reserve `_matchStrings` to `matchList.size` even though the existing code didn't do it. `emplace_back` would allow construction in place. Another optimization that didn't originally exist, but since we're touching the loop :)